lib/iex/lib/iex/evaluator.ex

```diff
@@ -82,6 +102,13 @@ defmodule IEx.Evaluator do
     end
   end
 
+  defp find_matched_variables(binding, var_prefix) do
+    for {var_name, _value} <- binding,
```

Comment:
Please make sure that variable names in bindings are atoms, they are not always atoms in bindings because of hygiene, and then use `Atom.to_string/1`.